source_gen/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## 0.9.2
 
 * Avoid using the AST analyzer model from `LibraryReader`.
+* Don't create part files with no corresponding library.
```

Comment:
are we worried about the potential bad interaction with already published versions of `build_resolvers` where the part directives will appear to be missing?

Reply:
No - the build_resolvers package was already doing the right thing actually (returning a non-existing  source instead of null)